upgrades/core/markets.json

```diff
@@ -0,0 +1,4056 @@
+[
```

Comment:
Is there a reason we need another core market json? We should ideally just have a single central one.

Reply:
gonna be a footgun trying to maintain the same file in different places.

Reply:
this is the markets as a list - i can just generate them on the fly